xtext.cpp

```diff
@@ -577,7 +581,7 @@ class XText {
             luaL_typerror(m_L, -1, "cdata");
         }
         // Get the usable (for Harfbuzz) FT_Face object
-        FT_Face face = *(FT_Face *)lua_topointer(m_L, -1);
+        FT_Size size = *(FT_Size *)lua_topointer(m_L, -1);
```

Comment:
The comment above should be adapted for the change.
I wouldn't mine in here some small comments explaining what FT_Size is and how it relates to FT_Face.

(Also, I would naively expect a variable named "size" to be a number :) while here it is an FT internal object, so dunno if we could give it a better name like `sized_face` ? Not insisting on that, if you prefer the straightforward `FT_Size size` :) but it needs a comment so that we understand early it's not a number.)

Reply:
Amended.